Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps.yml

```diff
@@ -773,6 +773,2230 @@ script:
+      - type: String
+        contextPath: AzureDevOps.PullRequestReviewer.imageUrl
+        description: Link to the reviewers user image.
+  - name: azure-devops-pull-request-commit-list
+    description: Get the commits for the specified pull request.
+    arguments:
+      - name: organization_name
+        description: The name of the Azure DevOps organization. Default value will be config param, user can supply a different value.
+        required: false
+        isArray: false
+      - name: project_name
+        description: Project ID or project name. Default value will be config param, user can supply a different value.
+        required: false
+        isArray: false
+      - name: repository_id
+        description: The repository ID. Default value will be config param, user can supply a different value.
+        required: false
+        isArray: false
+      - name: pull_request_id
+        description: ID of the pull request. By using the azure-devops-pull-request-list command, you can obtain the ID.
+        required: true
+        isArray: false
+      - defaultValue: '50'
+        description: The number of results to retrieve. Minimum value is 1.
+        name: limit
+    outputs:
+      - type: Number
+        contextPath: AzureDevOps.Commit.count
+        description: Count of commits.
+      - type: String
+        contextPath: AzureDevOps.Commit.value.commitId
+        description: ID (SHA-1) of the commit.
+      - type: String
+        contextPath: AzureDevOps.Commit.value.author.name
+        description: Name of the commit author.
+      - type: String
+        contextPath: AzureDevOps.Commit.value.author.email
+        description: Email address of the commit author.
+      - type: Date
+        contextPath: AzureDevOps.Commit.value.author.date
+        description: Date of the commit operation.
+      - type: String
+        contextPath: AzureDevOps.Commit.value.committer.name
+        description: Name of the commit committer.
+      - type: String
+        contextPath: AzureDevOps.Commit.value.committer.email
+        description: Email address of the commit committer.
+      - type: Date
+        contextPath: AzureDevOps.Commit.value.committer.date
+        description: Date of the commit operation.
+      - type: String
+        contextPath: AzureDevOps.Commit.value.comment
+        description: Comment or message of the commit.
+      - type: String
+        contextPath: AzureDevOps.Commit.value.url
+        description: REST URL for this resource.
+  - name: azure-devops-commit-list
+    description: Retrieve git commits for a project.
+    arguments:
+      - name: organization_name
+        description: The name of the Azure DevOps organization. Default value will be config param, user can supply a different value.
+        required: false
+        isArray: false
+      - name: project_name
+        description: Project ID or project name. Default value will be config param, user can supply a different value.
```

Comment:
clarify `will be config param, user can supply a different value` in all places please.